python_modules/dagster/dagster/core/selector/subset_selector.py

```diff
@@ -334,3 +353,44 @@ def parse_step_selection(step_deps, step_selection):
         steps_set.update(subset)
 
     return frozenset(steps_set)
+
+
+def parse_asset_selection(
+    assets_defs: Sequence["AssetsDefinition"], asset_selection: Sequence[str]
+) -> AbstractSet["AssetKey"]:
+    """Take in a set of AssetsDefinition objects and a list of asset selection queries and return
```

Comment:
Nitpick: this comment is I think fully redundant with the type annotations. Might make sense to leave it out or replace it with something like "Find assets that match the given selection query".